tests/functional/actions.js

```diff
@@ -2,36 +2,36 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */ 
 
-const elements = require('./elements.js');
+const { elementState } = require('./elements.js');
 const vpn = require('./helper.js');
- 
-const setConnectionChangeNotification = async (status) => {
-    return await vpn.setSetting('connection-change-notification', status ? "true" : "false");
-}
 
-const setServerSwitchNotification = async (status) => {    
-    return await vpn.setSetting('server-switch-notification', status ? "true" : "false");
-}
+const settings = {
+    setServerSwitchNotification: async (status) => {    
+        return await vpn.setSetting('server-switch-notification', status ? "true" : "false");
+    },
 
-const selectCountryFromList = async (countryId) => {
-  await vpn.setElementProperty(elements.SERVER_COUNTRY_VIEW, 'contentY', 'i', parseInt(await vpn.getElementProperty(countryId, 'y')));
-  await vpn.wait()
+    setConnectionChangeNotification: async (status) => {
+        return await vpn.setSetting('connection-change-notification', status ? "true" : "false");
+    }
 }
 
-const selectCityFromList = async (cityId, countryId) => {
-  await vpn.setElementProperty(elements.SERVER_COUNTRY_VIEW, 'contentY', 'i', parseInt(await vpn.getElementProperty(cityId, 'y')) + parseInt(await vpn.getElementProperty(countryId, 'y')));
-  await vpn.wait()
+const serverList = {
+    selectCityFromList: async (cityId, countryId) => {
+        await vpn.wait()
+        await vpn.setElementProperty(elementState.COUNTRY_VIEW, 'contentY', 'i', parseInt(await vpn.getElementProperty(cityId, 'y')) + parseInt(await vpn.getElementProperty(countryId, 'y')));
+        await vpn.wait()
```

Comment:
same here. We should wait for the end of the animation of to check if the 'contentY' is correctly updated.